### ISSUE TITLE
G32 Z-probe lowers z-probe outside probing boundaries

### ISSUE THREAD

**Issue by strips** [repetier/Repetier-Firmware#605]
I get a G32 leveling failed! 

After a G28, G32 does not move X to Z_PROBE_X1. The bed is raised and IR does not hit bed. It's a near collision but you seem to have some failsafe and it stops and then moves to Z_PROBE_X1. This is a bad thing. It should have moved to Z_PROBE_X1 before raising bed. Doing it after is dangerous as it has already gone too far.

My Z-probe needs to move X20 to hit the bed. 

Z_PROBE_X1/3 = 20
I also have ENDSTOP_X_BACK_ON_HOME = 2

Video: https://youtu.be/0MEPeWJ-Bz8

Regards
Stian

**Comment by strips** [comment]
This might be something altogether different. When I did a G1 X70 before G32 it raised the bed but seemed to ignore the Z-probe triggering. Going up to the same Z  where it failed the last time and then dragged the nozzle over the bed to X20. 

Still it looks like something is out of order.

**Comment by strips** [comment]
Just noticed the error message: Error:z-probe triggered before starting probing.

**Comment by repetier** [comment]
There are hundreds of different z probe routines depending on settings so that is always hard to follow without any information.

After G28 you are most probably too close to bed as probe already is high there. You need to start from a height where probe is not triggered and also at a xy position where activating probe is possible. Then G32 does normally complete.
